sw/device/tests/aes/aes_test.c

```diff
@@ -0,0 +1,87 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Which parts of `string.h` are you using?

Reply:
Nothing actually, good point. Thanks.